common/src/main/java/com/frostwire/bittorrent/BTEngine.java

```diff
@@ -258,6 +258,10 @@ public void download(File torrent, File saveDir, boolean[] selection) {
     }
 
     public void download(TorrentInfo ti, File saveDir, boolean[] selection, List<TcpEndpoint> peers) {
+        download(ti, saveDir, selection, peers, false);
```

Comment:
shouldn't this parameter be the current config value instead of defaulting to `false`?